<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>histogramwndClass</class>
 <widget class="QWidget" name="histogramwndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>histogrammwnd</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="7">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="spinBox_Low">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_High">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="PreviewButton">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QRadioButton" name="LinearButton">
     <property name="text">
      <string>lin.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QRadioButton" name="LogButton">
     <property name="text">
      <string>log.</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="AutoHist">
     <property name="text">
      <string>auto</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>PreviewButton</tabstop>
  <tabstop>LinearButton</tabstop>
  <tabstop>LogButton</tabstop>
  <tabstop>AutoHist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
